### PR TITLE
CSM-2246 - Update EsReview based on design feedback

### PR DESCRIPTION
**Pull request by RyanMulready**
### 🔗 Linked issue

- https://energysage.atlassian.net/browse/CSM-2246

### ❓ Type of change

- [X] 📖 Documentation (updates to the documentation or readme)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [X] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description

- Removed margin and fixed heights on `EsReview` and moved it to wrapper in documentation
- Removed horizontal scroll from documentation
- Fixed an issue where sidebar in `EsReviewModal` was reliant upon `body-bg` instead of modal bg

### 🥼 Testing

- Updated test snapshots

### 📝 Checklist

- [X] I have linked an issue or discussion.
- [X] I have updated the documentation accordingly.
- [X] I have documented testing approach

### CODE REVIEW

File: es-design-system/pages/organisms/es-reviews-list.vue [hunk 1]
```diff
@@ -162,12 +133,18 @@ export default {
 @import '~@energysage/es-bs-base/scss/includes';
 
 .review-wrapper {
-    width: 450px;
+    border-right: 1px solid $border-color;
+    width: 470px;
```
Comment (gigiwargin) [inline es-design-system/pages/organisms/es-reviews-list.vue:137]: On my screen, the third review is scrunched to the side (screenshot below). I would expect that the width of each of the reviews would be responsive to viewport width. 
Screen Shot 2022-12-09 at 3 49 10 PM

This change worked for me, but I'm not sure if that's exactly what we're going for 🤷‍♀️ 
```suggestion
    width: 30%;
```

**Review by gigiwargin — COMMENTED** [review]
These changes look great, especially on mobile!

A couple comments inline, but otherwise nice job 🥳

File: es-design-system/pages/organisms/es-reviews-list.vue (hunk 1 above)
Comment (RyanMulready) [reply es-design-system/pages/organisms/es-reviews-list.vue]: Good call out. Looking at it more there's no reason this should just be a `b-row` b-col` layout now. I've updated the docs.

**Review by gigiwargin — APPROVED** [review]
Spacing looks great now! Nice job :)

**Pull request merged**